### ISSUE TITLE
List all the models names

### ISSUE THREAD

**Issue by yupbank** [tensorflow/serving#795]
Hey, right now we can query model by their name, but we can we query the serving binary to have a list of all models available?

**Comment by chrisolston** [comment]
Thanks for proposing a PR! We'll take a look at it.

**Comment by ymodak** [comment]
Closing this issue since PR has been proposed. Stay tuned to this [issue](https://github.com/tensorflow/serving/pull/797) for updates.

**Comment by kesh-me** [comment]
@ymodak The referenced issue was already closed on Mar 29, 2018. This should be re-opened.

**Comment by yupbank** [comment]
should we just reopen my original pr https://github.com/tensorflow/serving/pull/797 and move forward with that ? list model names that is alive should be sufficient ?

**Comment by kesh-me** [comment]
@yupbank Loaded model names is limited in information. I prefer having loaded model versions and other metdata defined in ModelServerConfig proto to be part of this service's response.

**Comment by yupbank** [comment]
yeah, but you can get the versions and other metadata by using the model name

**Comment by rola93** [comment]
is there any news on this?

**Comment by kesh-me** [comment]
@yupbank +1 to open up your PR.

**Comment by stefan-falk** [comment]
Any progress on this https://github.com/tensorflow/serving/pull/1552  ?

**Comment by Izzette** [comment]
Beep-boop have the maintainers had a chance to take a look at the PR?

**Comment by bharatjetti** [comment]
Hi @yupbank  Thank you so much for showing interest! We truly appreciate your enthusiasm and support.
We regret to inform you that we are not accepting new implementation requests or feature additions at this time. Please feel free to fork the repository and submit pull requests if you have any suggestions for enhancements, problem fixes, or ideas. We’d be more than happy to review and collaborate! If you would like to contribute , please follow our [CONTRIBUTION GUIDELINES](https://github.com/tensorflow/serving/blob/master/CONTRIBUTING.md) here and submit a pull request.
Thank you again for your interest, and we look forward to seeing your contributions!

**Comment by jeelania10** [comment]
Closing this due to inactivity. Please feel free to reopen and post your comments (if you still have queries on this). Thank you!
